src/Settings.php

```diff
@@ -249,27 +258,29 @@ public function add_settings() {
 				'date_format' => [
 					'type'            => 'text',
 					'label'           => esc_html__( 'Date format', 'tribe-ext-daystrip' ),
-					'tooltip'         => sprintf( esc_html__( 'Examples: %1$s - 1, %2$s - 01, %3$s - 1st, %4$s - hide', 'tribe-ext-daystrip' ),
+					'tooltip'         => sprintf( esc_html__( 'Examples: %1$s - "1", %2$s - "01", %3$s - "1st", %4$s (or empty) - hide', 'tribe-ext-daystrip' ),
 					                              '<code>j</code>',
 					                              '<code>d</code>',
 					                              '<code>jS</code>',
 					                              '<code>0</code>',
 					),
 					'validation_type' => 'alpha_numeric',
+					'can_be_empty'    => true,
 					'size'            => 'small',
 					'default'         => 'j',
 				],
 				'month_format' => [
 					'type'            => 'text',
 					'label'           => esc_html__( 'Month format', 'tribe-ext-daystrip' ),
-					'tooltip'         => sprintf( esc_html__( 'Examples: %1$s - Jan., %2$s - January, %3$s - 01, %4$s - 1, %5$s - hide', 'tribe-ext-daystrip' ),
+					'tooltip'         => sprintf( esc_html__( 'Examples: %1$s - "Jan", %2$s - "January", %3$s - "01", %4$s - "1", %5$s (or empty) - hide', 'tribe-ext-daystrip' ),
```

Comment:
```suggestion
					'tooltip'         => sprintf( esc_html__( 'Examples: %1$s - "Jan.", %2$s - "January", %3$s - "01", %4$s - "1", %5$s (or empty) - hide', 'tribe-ext-daystrip' ),
```
Based on the previous copy, we seem to be missing period here?